test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,67 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+const md5 = require('md5');
+const data = require('./data.json');
+
+const url = 'https://api.github.com/users';
+const userName = 'aperdomob';
+
+chai.use(chaiSubset);
+
+describe('Github Api Repositories Test', () => {
+  let responseData;
+  let repoName;
+  it('GET test', async () => {
+    const response = await axios.get(`${url}/${userName}`);
+
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data.name).to.equal(data.name);
+    expect(response.data.location).to.equal(data.location);
+    expect(response.data.company).to.equal(data.company);
+    responseData = response;
+  });
+
+  it('search repository', async () => {
+    const responseTwo = await axios.get(`${responseData.data.repos_url}`);
+    const repository = responseTwo.data.find((repo) => repo.name === 'jasmine-json-report');
+
+    expect(responseTwo.status).to.equal(StatusCodes.OK);
+    expect(repository.name).to.equal('jasmine-json-report');
+    expect(repository.private).to.equal(false);
+    expect(repository.description).to.equal('A Simple Jasmine JSON Report');
+    repoName = repository;
+  });
+
+  it('Dowload repository', async () => {
+    const dowload = await axios.get(`${repoName.svn_url}/archive/${repoName.default_branch}.zip`);
+
+    expect(dowload.status).to.equal(StatusCodes.OK);
+    expect(dowload.headers['content-type']).to.equal('application/zip');
+  });
+
+  let readme;
+  it('Repository files', async () => {
+    const file = await axios.get(`${repoName.url}/contents`);
+    const dataFile = file.data.find((content) => content.name === 'README.md');
```

Comment:
you can assign directly to the `readme` variable, instead of declaring a new const here